.appveyor.yml

```diff
@@ -0,0 +1,50 @@
+# Version format
+version: "{build}"
+
+image: Visual Studio 2015
+
+# Environment variables
+environment:
+  GOPATH: c:\gopath
+  GVM_GO_VERSION: 1.10.3
```

Comment:
hm... we rarely update appveyor configs. Sometimes running with old version until things break due to someone using new APIs.

I'm considering to introduce special versions `stable` and 'latest'. The `stable` version is highest stable go relesae, while `latest` can include go beta/rc releases.

Reply:
Sounds like a good idea.